### ISSUE TITLE
SteamOS installer destroys partition table on other drive

### ISSUE THREAD

**Issue by dbenoy** [ValveSoftware/SteamOS#623]
#### Steps for reproducing this issue:

1. Run the SteamOS installer in expert mode on a system with both a SATA drive and a NVME drive
2. Instruct the installer to partition the SATA drive but leave the NVME drive alone
3. Continue with the install and watch as your NVME drive partition table recieves the changes you asked to be put on the SATA drive.

Found this out the hard way.

**Comment by dbenoy** [comment]
I was able to recover my partitions, so it looks like this just affects the partition table and not the volumes themselves.

Also for others stumbling upon this problem, here's a workaround: Before getting to the 'detect disks' phase, drop to a shell with ctrl-alt-F2, find the 'nvme.ko' file in /lib/modules and delete it, then try detecting the disks and it should fail to find your drives, and most likely will be unable to interact at all with nvme at that point.

**Comment by dbenoy** [comment]
Is this issue being looked at? I'd hate for someone else to fall victim to this problem and end up losing all their data.

**Comment by meLox1337** [comment]
No, I don't think it's being looked at @dbenoy I had the same problem affecting me once with a different though upgrading from win7 to win10 the issue was never looked at by Microsoft either but then I discovered that there was an issue with a driver not executing properly and a raid setup was initialized by my system without my consent or command. rather unplug all storage drives except the one you want to install the os on in the future that's what I do.
